pkg/config/config.go

```diff
@@ -134,13 +137,29 @@ func (cfg *Config) SetupLogger() {
 	ctrlrt.SetLogger(zap.New(zap.UseFlagOptions(&zapOptions)))
 }
 
+// PopulateAccountIdIfMissing uses sts GetCallerIdentity API to find
+// AWS AccountId when Config.AccountId is empty
+func (cfg *Config) PopulateAccountIdIfMissing() error {
```

Comment:
Please change `Id` to `ID` everywhere applicable
```suggestion
func (cfg *Config) PopulateAccountIDIfMissing() error {
```